dGame/dGameMessages/PropertyData.h

```diff
@@ -0,0 +1,39 @@
+#pragma once
+
+#include "Entity.h"
+
+
+struct PropertyPersonalData final {
+	bool IsBestFriend;
```

Comment:
Generally our members of structs and classes follow `m_FieldName`for member fields, is there a particular reason this isnt done here?

Reply:
we've followed this strictly for members of classes, but not for structs